dashboard/src/app/(authenticated)/externalEventDef/[name]/components/ExternalEventDef.tsx

```diff
@@ -27,7 +28,7 @@ export const ExternalEventDef: FC<Props> = ({ spec }) => {
   const { tenantId } = useWhoAmI()
 
   const { isPending, data, hasNextPage, fetchNextPage } = useInfiniteQuery({
-    queryKey: ['taskRun', tenantId, 10, createdAfter, createdBefore, isClaimed],
+    queryKey: ['externalEvent', tenantId, 10, createdAfter, createdBefore, isClaimed],
```

Comment:
What that 10 means?

Reply:
Not sure, ill remove it, it was copied from somewhere else. Maybe its needed in that other example so I'm not going to remove it in the other instance.